components/layout_2020/display_list/stacking_context.rs

```diff
@@ -279,18 +330,21 @@ impl StackingContext {
             initializing_fragment_style: None,
             context_type: StackingContextType::Real,
             fragments: vec![],
-            stacking_contexts: vec![],
-            float_stacking_contexts: vec![],
+            stealable_children: vec![],
+            float_stacking_containers: vec![],
+            inline_stacking_containers: vec![],
         }
     }
 
     /// Add a child stacking context to this stacking context.
     fn add_stacking_context(&mut self, stacking_context: StackingContext) {
-        if stacking_context.context_type == StackingContextType::PseudoFloat {
-            self.float_stacking_contexts.push(stacking_context);
-        } else {
-            self.stacking_contexts.push(stacking_context);
+        match stacking_context.context_type {
+            StackingContextType::Real => &mut self.stealable_children,
+            StackingContextType::PseudoPositioned => &mut self.stealable_children,
```

Comment:
Maybe this is a good moment to standardize on "stacking container" and rename these to be something like "PositionedStackingContainer" or whatever works for you.

Reply:
done in 8e60b20ca11c2a3d755e21931661c14d108f9261